util/cli/queue.go

```diff
@@ -0,0 +1,123 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/src-d/lookout"
+	"github.com/src-d/lookout/util/ctxlog"
+	"github.com/src-d/lookout/util/queuehelper"
+	"gopkg.in/src-d/go-queue.v1"
+	_ "gopkg.in/src-d/go-queue.v1/amqp"
+	_ "gopkg.in/src-d/go-queue.v1/memory"
+)
+
+// queueWindow defines the maximum number of unacknowledged jobs that the
+// queue iterator will allow to retrieve. A number of 1 will mean that each
+// job is processed sequentially
+// TODO: because the eventHandler is called synchronously, a value greater than
```

Comment:
nit pick, but it's usually a good idea to save reader some time blaming the changes and put `TODO(<name>) `

Reply:
I agree, but that seems to me that applies to when you have some half-baked code that you plan to add later.
In this case, however, I wanted to mark future work to be done by whoever, with a detailed explanation of what is missing and how to proceed. I'd prefer not to mark this as reserved for me, when anybody could implement the missing functionality.

Reply:
I see the confusion you are having - it's quite common. This comment *does not claim* the issue ownership, it just attributes it to particular author who has the best context over it.
 
Same rules of code ownership are applied as anywhere else, so anyone can do that.

Please read more about it - it's very well explained in https://google.github.io/styleguide/cppguide.html#TODO_Comments

That is a normal engineering best-practice and it's up to you of course to follow it or not.
I just suggest to follow.

Reply:
done in d668151